src/Domain/CoordinateMaps/TimeDependent/ShapeMapTransitionFunctions/SphereTransition.cpp

```diff
@@ -55,10 +56,26 @@ std::optional<double> SphereTransition::original_radius_over_radius(
   }
   const double original_radius = (mag + radial_distortion * b_) / denom;
 
-  return (original_radius + eps_) >= r_min_ and
-                 (original_radius - eps_) <= r_max_
-             ? std::optional<double>{original_radius / mag}
-             : std::nullopt;
+  // If we are within r_min and r_max, doesn't matter if we are reverse
+  // or extending, we just return the value we calculated. If we are extending,
+  // then if we are reverse and the point is outside r_max or we aren't reversed
+  // and the point is inside r_min, then we return a simplified formula since
+  // the transition function is 1 in this region. If we are extending, but the
+  // above conditions aren't true, then we are in the region where the
+  // transition function is 0, so we return 1.0. Otherwise we return nullopt.
+  if ((original_radius + eps_) >= r_min_ and
+      (original_radius - eps_) <= r_max_) {
+    return std::optional<double>{original_radius / mag};
+  } else if (extend_) {
+    // a_ being positive is a sentinel for reverse (see constructor)
+    if ((a_ > 0.0 and mag > r_max_) or (a_ < 0.0 and mag < r_min_)) {
+      return {1.0 + radial_distortion / mag};
```

Comment:
Do you have a test for the inverse in this regime, which maps the point back and forth through the wedge? We should have that explicitly somewhere to guarantee that the wedge+shape map and its inverse work outside of the wedge